django_mongodb/utils.py

```diff
@@ -25,6 +26,58 @@ def check_django_compatability():
         )
 
 
+def parse(
+    uri, engine=None, conn_max_age=None, conn_health_checks=None, ssl_require=None, test=None
+):
+    """
+    Parse a MongoDB URI and return a dictionary of Django database
+    settings. This function is a wrapper around PyMongo's
+    ``pymongo.uri_parser.parse_uri()`` function that converts PyMongo's
+    settings dictionary into a Django database settings dictionary.
+    """
+    uri = parse_uri(uri)
+
+    host = None
+    port = None
+
+    if not engine:
```

Comment:
As far as I can think, this would be useful if something is subclassing our backend. Seems unlikely. In any case,  you could simply make `engine="django_mongodb"` the default kwarg if you want to keep it.